clear: add -x, -T and -V examples (#3684)
tldr-pages/tldr: pages/common/clear.md

> Clears the screen of the terminal.

- Clear the screen (equivalent to pressing Control-L in Bash shell):

`clear`

- Clear the screen but keep the terminal's scrollback buffer:

`clear -x`

> - Indicate the type of terminal to clean (normally this option is unnecessary because it is taken from the environment variable `TERM`):

Reviewer: Perhaps this will read better?
```suggestion
- Indicate the type of terminal to clean (defaults to the value of the environment variable `TERM`):
```

> `clear -T {type_of_terminal}`

Reviewer: I think you meant to have double `{{}}` here?
```suggestion
`clear -T {{type_of_terminal}}`
```

Author: Ups! I little slip. I will fixed it.

> - Clear the screen (equivalent to typing Control-L when using the bash shell):

Reviewer 2: (Unrelated to this PR, but I caught it now)
```suggestion
- Clear the screen (equivalent to pressing Control-L in Bash shell):
```

`clear`

> - Clear the screen but keep the terminal's scrollback buffer:

Reviewer 2: I'd replace `terminal's scrollback buffer` with something simpler.

Author: What would you propose? Now I don't know something simpler to explain it, but I will think about it.

Reviewer 2: Sounds way too technical, but understandable. If we don't come up with anything, this can stay, it's not a problem.

Author: Yes, I am agree with your argument. May be... "Clear the screen and allow to scroll up to the previously used commands" or "Clear the screen and allow to scroll up to the the previous typed commands"?

Reviewer: I'd argue that we should keep it as-is. I'm not sure that it's any simpler if we change the language, and we'd risk being ambiguous if we were to change it.

Author: Nice! Thank you for your comment, @sbrl. I am going to mark it as resolved.

`clear -x`

- Indicate the type of terminal to clean (normally this option is unnecessary because it is taken from the environment variable `TERM`):

`clear -T {type_of_terminal}`

> - Show the version of `ncurses` used by `clear`.

Reviewer 2: Is this really necessary? Seems unneeded to absolute majority of the users.

Author: Yes, that's true. But it is a command with just a few parameters. Do you think it would be necessary to remove it?

Reviewer 2: You can wait for the opinions of others, but I'd keep only the useful examples. :wink:

Author: As we are waiting for more opinions in the above issue, we can wait.

Reviewer: There's no harm in including it, I guess - since there isn't anything else we can include on the  page.

Author: So could it being marked as resolved? Or should we wait for more opinions about this?

Reviewer 3: I think we could maintain the example and if we have problems exceeding the examples limit, then this example would be the first one to be removed.
